serialization/json_ld/context.json

```diff
@@ -0,0 +1,745 @@
+        }
+    },
+    "anonymizationMethodUsed": {
+        "@id": "dataset:anonymizationMethodUsed",
+        "@type": "xsd:string"
+    },
+    "assessedElement": {
+        "@id": "security:assessedElement",
+        "@type": "@id"
+    },
+    "attributionText": {
+        "@id": "software:attributionText",
+        "@type": "xsd:string"
+    },
+    "autonomyType": {
+        "@id": "ai:autonomyType",
+        "@type": "ai:PresenceType"
+    },
+    "begin": {
+        "@id": "core:begin",
+        "@type": "xsd:positiveInteger"
+    },
+    "build": "https://spdx.org/rdf/Build/",
+    "buildEndTime": {
+        "@id": "build:buildEndTime",
+        "@type": "core:DateTime"
+    },
+    "buildId": {
+        "@id": "build:buildId",
+        "@type": "xsd:string"
+    },
+    "buildStartTime": {
+        "@id": "build:buildStartTime",
+        "@type": "core:DateTime"
+    },
+    "buildType": {
+        "@id": "build:buildType",
+        "@type": "xsd:anyURI"
+    },
+    "builtTime": {
+        "@id": "core:builtTime",
+        "@type": "core:DateTime"
+    },
+    "byteRange": {
+        "@id": "software:byteRange",
+        "@type": "core:PositiveIntegerRange"
+    },
+    "catalogType": {
+        "@id": "security:catalogType",
+        "@type": "@vocab",
+        "@context": {
+            "@vocab": "security:ExploitCatalogType/"
+        }
+    },
+    "comment": {
+        "@id": "core:comment",
+        "@type": "xsd:string"
+    },
+    "completeness": {
+        "@id": "core:completeness",
+        "@type": "@vocab",
+        "@context": {
+            "@vocab": "core:RelationshipCompleteness/"
+        }
+    },
+    "concludedLicense": {
+        "@id": "software:concludedLicense",
+        "@type": "licensing:AnyLicenseInfo"
+    },
+    "conditionality": {
+        "@id": "software:conditionality",
+        "@type": "@vocab",
+        "@context": {
+            "@vocab": "software:DependencyConditionalityType/"
+        }
+    },
+    "confidentialityLevel": {
+        "@id": "dataset:confidentialityLevel",
+        "@type": "@vocab",
+        "@context": {
+            "@vocab": "dataset:ConfidentialityLevelType/"
+        }
+    },
+    "configSourceDigest": {
+        "@id": "build:configSourceDigest",
+        "@type": "core:Hash"
+    },
+    "configSourceEntrypoint": {
+        "@id": "build:configSourceEntrypoint",
+        "@type": "xsd:string"
+    },
+    "configSourceUri": {
+        "@id": "build:configSourceUri",
+        "@type": "xsd:anyURI"
+    },
+    "contentIdentifier": {
+        "@id": "software:contentIdentifier",
+        "@type": "xsd:anyURI"
+    },
+    "contentType": {
+        "@id": "core:contentType",
+        "@type": "core:MediaType"
+    },
+    "context": {
+        "@id": "core:context",
+        "@type": "xsd:string"
+    },
+    "copyrightText": {
+        "@id": "software:copyrightText",
+        "@type": "xsd:string"
+    },
+    "core": "https://spdx.org/rdf/Core/",
```

Comment:
The URIs should follow the patterns defined in `rdf.md`
Please include the `/v3/` path part everywhere.

Reply:
This is a problem with the spec-parser. I have opened a PR: https://github.com/spdx/spec-parser/pull/60